Tests/PureeTests/Output/BufferedOutputDispatchQueueTests.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import XCTest
+@testable import Puree
+
+private func makeLog() -> LogEntry {
+    return LogEntry(tag: "foo", date: Date())
+}
+
+class BufferedOutputDispatchQueueTests: XCTestCase {
+
+    func testFlushInterval() {
+        let exp = expectation(description: #function)
+
+        // Run the test on a background queue
+        let dispatchQueue = DispatchQueue(label: "com.cookpad.Puree.Logger", qos: .background)
+        dispatchQueue.async {
+
```

Comment:
unnecessary

Reply:
I think the idea here is to make sure that timer works correctly regardless of the thread and this needs to be here

Reply:
I mentioned about a blank line.